modules/apps/frontend-theme/frontend-theme-admin/src/css/portlet/_topper.scss

```diff
@@ -1,3 +1,9 @@
 .portlet {
 	position: relative;
+
+	.taglib-workflow-status {
+		.workflow-version {
+			color: #6b6c7e !important;
+		}
+	}
```

Comment:
Hi @icgsbr. We can make this into our module and not change the frontend-theme module. I was able to get it working by doing the following in this file:

`/home/me/dev/projects/liferay-portal/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss`

**_general.scss:**
```
.portal-popup,
.portlet-forms-admin {

 *** existing code***
 
 .taglib-workflow-status {
		.workflow-version {
			color: #6b6c7e;
		}
	}
}
```
 Please, could you check if it works for you?
Thanks!

Reply:
Done! Thanks for the help!